docs/database/database-spans.md

```diff
@@ -478,6 +480,39 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Propagating context to databases
+
+**Status**: [Development][DocumentStatus]
+
+Instrumentations SHOULD propagate the context information to the SQL queries following [sqlcommenter](https://google.github.io/sqlcommenter/spec/).
+
+### Recommended attributes
+
+| Attribute              | Type   | Description                           | Require level     | Stability                                                      |
+|------------------------|--------|---------------------------------------|-------------------|----------------------------------------------------------------|
+| `baggage.service.name` | string | Logical name of the service [1]       | `Required`        | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
If `traceparent` is present I do not think that service.name is needed. Isn't it the best way to have one or another?

Do we even need to define "require level"? Maybe we should just describe the meaning of each sqlcommenter attribute?

Reply:
> If `traceparent` is present I do not think that service.name is needed. Isn't it the best way to have one or another?

Updated, and I keep the `Require level` to make it as an annotation as well. How do you think about it?

Reply:
> If `traceparent` is present I do not think that service.name is needed. Isn't it the best way to have one or another?

Updated, and I keep the `Require level` to make it as an annotation as well. What do you think about it?

Reply:
I am not sure if we need even the require level. I think these are just different propagation options :wink: But we can keep it for now as I think there are other more important comments to be resolved/addressed.

Reply:
> If traceparent is present I do not think that service.name is needed. 

Even if `traceparent` is present, `service.name` is still useful. Exact correlation using `trace_id` requires both a database sample and the full trace for the same request, but because of sampling, many traces and DB executions are missed. Since the chance of both being captured is low (`p_trace * p_db_sample`), including `service.name` helps support approximate correlation when exact matching isn’t possible.